### PR TITLE
Docs for coding convention and component customizations

### LINKED ISSUE #197: Describe the Coding Standards + component customization
## What is the motivation for adding/enhancing this feature?

We've got some pretty cool coding standards that are pretty easy to get like:

```
import MRelatedProducts from 'theme/components/molecules/m-related-products';
import OProductDetails from 'theme/components/organisms/o-product-details';
import AImagesGrid from 'theme/components/atoms/a-images-grid';
```

but they're not documented. Can we just create a short *.md file and link it to the main README with the description?

The other question ppl. will have immediately after releasing DPR is "How to customize the look and feel". There is plenty of options for this in Storefront UI - but I think we should have it described in a *.md file linked to main README.

### PR DESCRIPTION
**Pull request by psmyrek**
### Related Issues

Closes #197 

### Short Description and Why It's Useful

This change adds short description of our coding standards and conventions we are using in our repo.

### Screenshots of Visual Changes before/after (If There Are Any)

No visual changes in app.

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: CODING_STANDARDS.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Coding standards & convention
+
+We have decided to create and follow some guidelines and best practices during development of Capybara theme. There are a few core goals that we stick to because we belive they improve our cooperation and readability of source code we've created.
```
Comment (qiqqq) [inline CODING_STANDARDS.md]: Type: is "belive" and should be "believe"

File: CODING_STANDARDS.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## Atomic Design rules
+
+[Atomic Design](https://atomicdesign.bradfrost.com/) is a methodology for creating user interfaces in structured and hierarchical manner. It is composed from 5 *elements* (or *stages*): atoms, molecules, organisms, templates and pages. Here in Capybara we have slightly adapted Atomic Design rules to our needs and we are using all of them except templates, because we've not found any use case for them. Each of those elements play a key role in designing our interface. In the `/components` folder we created a dedicated sub-folder for atoms, molecules and organisms. Each `.vue` file inside them has a name prefixed by `a-` for atoms, `m-` for molecules and `o-` for organisms. Similarly, the name for every our component follow the same naming convention: it is prefixed by `A` for atoms, `M` for molecules and `O` for organisms. This approach helps us categorize and figure out very fast what kind of component we see in the markup. Pages on the other hand are located in dedicated `/pages` folder. Because they are located in a separate location and they are not used within templates like ordinary components, they do not need to be prefixed.
```
Comment (qiqqq) [inline CODING_STANDARDS.md]: Typo: is "composed from" and should be "composed of"

**Comment by qiqqq** [comment]
It is simple to read and understand. Good work!

File: CODING_STANDARDS.md (hunk 1 above)
Comment (psmyrek) [reply CODING_STANDARDS.md]: Fixed.

File: CODING_STANDARDS.md (hunk 2 above)
Comment (psmyrek) [reply CODING_STANDARDS.md]: Fixed.

**Pull request merged**